### ISSUE TITLE
Adding support for GigaSpeech for ASR

### ISSUE THREAD

**Issue by AgaDob** [speechbrain/speechbrain#1198]
Hello, I was wondering whether there are any plans to add support for GigaSpeech for ASR in the future? :)

**Comment by TParcollet** [comment]
Yes, but we do not have the bandwidth unfortunately. I have downloaded the datasets, but now we need to create the data_preparation script ... If you feel like this is something you could do, we would be happy to provide the recipes afterward and even train some models.

**Comment by mravanelli** [comment]
Yes, the help from the community is welcome here! ;)

**Comment by TParcollet** [comment]
I quickly looked, and it wouldn't be that hard. I believe that the data_prep script is a matter of 2 hours of work. But I don't have this time right now :/
